packages/sandpack-core/src/resolver/utils/pkg-json.ts

```diff
@@ -68,5 +50,23 @@ export function processPackageJSON(
     }
   }
 
+  // load exports if it's not the root pkg.json
+  if (content.exports && pkgRoot !== '/') {
```

Comment:
Note that likely you only need to still process alias fields - but main fields don't have to be processed.

Reply:
You're right good catch, updated it accordingly